source/includes/gcp/_health_checks.md

```diff
@@ -1,5 +1,151 @@
 ### Health checks
 
+<!-------------------- List HEALTH CHECK -------------------->
+#### List health checks
+
+```shell
+curl -X GET \
+  -H 'MC-Api-key: your_api_key'
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/healthchecks"
+  # The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "checkIntervalSec": 10,
+      "creationTimestamp": "2019-09-18T12:39:48.982-07:00",
+      "description": "",
+      "healthyThreshold": 2,
+      "id": "5930212998788364011",
+      "kind": "compute#healthCheck",
+      "name": "healthcheckname",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/healthChecks/healthcheckname",
+      "httpHealthCheck": {
+        "host": "",
+        "port": 80,
+        "portName": "http",
+        "proxyHeader": "NONE",
+        "requestPath": "/"
+      },
+      "timeoutSec": 5,
+      "type": "HTTP",
+      "unhealthyThreshold": 3,
+      "portNumber": "80"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>
+   GET /service/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/healthchecks
+</code>
+
+Retrieve a list of all healtch checks
+
+Attributes | &nbsp;
+------- | -----------
+`checkIntervalSec`<br/>*int* | How often (in seconds) to send a health check. The default value is 5 seconds
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format
+`description` <br/>*string* | An optional description of this resource. Provide this property when you create the resource.
+`healthyThreshold`ç | A so-far unhealthy instance will be marked healthy after this many consecutive successes. The default value is 2.
+`httpHealthCheck` <br/>*object* | object
```

Comment:
Maybe give is a meaningful description ?
`When the type is set to HTTP, this object represents the health check`

And for the one below, 
`When the type is set to HTTPS, this object represents the health check`

Reply:
done